submissions/mariamPopoola/database/query.sql

```diff
@@ -0,0 +1,30 @@
+-- What is the total number of passengers who survived?
+
+SELECT COUNT(*) FROM accident_cases WHERE survive = 1;
+290
```

Comment:
This is not correct and it is most likely because you din not import thr correct number of contacts, The total should be 891. Import again and rerun your query.